src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,113 @@
+
+Setting Up Your Project Repository
+----------------------------------
+
+Fork the repository with the `starter code <https://github.com/LaunchCodeEducation/Launch-Checklist-Form/>`_ to your personal Github profile and clone the repository to the directory where you are keeping your assignments for the class.
+
+.. warning::
+
+   When updating styles to indicate whether the shuttle is ready for launch, do not modify ``styles.css``!
+
+To get started, navigate to the directory with your copy of the starter code. Open ``index.html`` with Firefox to verify that your starter code is working.
+
+When you open ``index.html``, you should see the Launch Checklist form with a rectangle above it for the mission destination and a rectangle below it that simply says "Awaiting Information Before Launch".
+
+.. figure:: figures/form-starting-point.png
+   :alt: Image showing the form and the box stating that more information is needed before launch.
+
+Adding Validation
+-----------------
+
+Adding Alerts
+^^^^^^^^^^^^^
+
+First, let's add the validation to notify the user if they have forgotten to enter a value for any one of the fields.
+
+This process is going to look similar to the :ref:`validation section <javascript-validation>` in the chapter on forms. 
+Make sure to use ``preventDefault()`` and an alert to notify the user that all fields are required.
+
+You also want to make sure that the user has entered valid info for each of the fields.
+Valid information for the fields means that the user has entered a value that is easily converted to the correct data type for our fellow engineers.
+The pilot and co-pilot names should be strings and the fuel level and cargo weight should be numbers.
+
+.. note:: 
+
+   If you want to check if something is ``NaN``, you cannot use ``==`` or ``===``.
+   JavaScript does have a built-in method called ``isNaN(value)`` where it returns ``true`` if ``value`` is ``NaN`` and ``false`` if ``value`` is not ``NaN``.
+
+Updating Shuttle Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The list of shuttle requirements, the ``div`` with the id ``faultyItems``, should be updated if something is not ready for launch. 
+Using template literals, update the ``li`` elements ``pilotStatus`` and ``copilotStatus`` to include the pilot's name and the co-pilot's name.
+
+If the user submits a fuel level that is too low (less than 10,000 liters), change the list to ``visible`` with an updated fuel status stating that there is not enough fuel for the journey.
+The text of the ``h2`` element, ``launchStatus``, should also change to red and "Shuttle not ready for launch".
+
+If the user submits a cargo mass that is too heavy (more than 10,000 kilograms), change the list to ``visible`` with an updated cargo status stating that there is too much mass for the shuttle to take off.
+The text of ``launchStatus`` should also change to red and "Shuttle not ready for launch".
```

Comment:
change to red and read

Reply:
This is the same idea as above.  'Red' is appropriate, I think.